### PR TITLE
Translate and update tutorials

### PR DESCRIPTION
**Pull request by eatyourgreens**

### CODE REVIEW

**Comment by eatyourgreens** [comment]
zooniverse/Panoptes#2392 stops us from creating translations of tutorials at the moment. Once that's fixed, this should work without any other changes.

**Comment by eatyourgreens** [comment]
Added a workaround for zooniverse/Panoptes#2392

@simoneduca this is ready to go now.

**Comment by simoneduca** [comment]
Having a look now

File: src/components/Tutorial.jsx [hunk 1]
```diff
@@ -13,8 +13,12 @@ options.log = (test) => {
 function Tutorial(props) {
   const { contents } = props;
   const tutorial = contents.original || { steps: [] };
+  const translation = contents.translation || { steps: [] };
   const steps = [];
-  tutorial.steps && tutorial.steps.map((step, key) => steps.push(<p key={key}><b>{key}</b> {step.content}</p>));
+  tutorial.steps && tutorial.steps.map((step, key) => {
+    steps.push(<p data-translation-key={`steps.${key}.content`} key={key}><b>{key}</b> {step.content}</p>);
+    translation.steps[key] && steps.push(<p key={`translation-${key}`}><b>translation</b> {translation.steps[key].content}</p>);
+  });
```
Comment (simoneduca) [inline src/components/Tutorial.jsx]: Shouldn't we support translation of the tutorial's title as well?

**Comment by simoneduca** [comment]
Works as expected: I can translate a tutorial step and it shows correctly under Translation header.
Not a blocking change, but shouldn't we enable translation of tut's title and add markdown support. Let me know if you prefer separate PRs for those and I'll action them.

File: src/components/Tutorial.jsx (hunk 1 above)
Comment (eatyourgreens) [inline src/components/Tutorial.jsx]: No harm in translating it, I suppose, but tutorial titles are only displayed in the project builder.

**Comment by eatyourgreens** [comment]
Good point about markdown. I'll add it.

File: src/components/Tutorial.jsx (hunk 1 above)
Comment (simoneduca) [inline src/components/Tutorial.jsx]: Happy to leave then.

**Comment by eatyourgreens** [comment]
I've added markdown for tutorial step content.

**Comment by simoneduca** [comment]
LGTM. However, I've noticed something: if there is more than one tutorial and since only one tutorial can be associated to any given workflow, this line https://github.com/zooniverse/pandora/blob/master/src/components/ProjectDashboard.jsx#L50 will fail with `Cannot read property '0' of undefined`. Happy to merge this and open separate issue or for @eatyourgreens to fix it here.

**Comment by eatyourgreens** [comment]
@simoneduca how do I reproduce that? My test project on staging has 3 tutorials linked to workflow 1483 and isn't throwing any errors.

**Comment by simoneduca** [comment]
It looks like the problem is that Pandora creates a new tutorial when translating an existing one, identical to the original but with different ID.
screen shot 2017-07-14 at 10 52 15
This is how is rendered in PFE
screen shot 2017-07-14 at 10 33 49

PFE tries to cope with that by linking both tutorials to the workflow.
screen shot 2017-07-14 at 10 40 06

If you try to unlink one, than Pandora crashes because one of the existing tutorial is missing `workflow` in its `links`.

If you delete the translated tutorial from PFE, then you're just back to square one with no translation.

Maybe linked  to https://github.com/zooniverse/Panoptes-Front-End/issues/3598

Possible solution don't create new tutorial when one is translated. Investigating.

**Comment by eatyourgreens** [comment]
Translations of tutorials are just new tutorial resources, with a different language code, linked to the same workflow.

I'm not sure how you would translate a tutorial that isn't linked to a workflow. The workflow link is what ties the two resources together. Interesting question, though.

**Comment by simoneduca** [comment]
This is been clarified on Slack. LGTM.

**Pull request merged**